lucid-book-store/src/components/cart-wishlist/wishlist-card.jsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+
+const WishlistCard = () => {
+  return (
+    <>
```

Comment:
why use react fragment here , when you are returning a single div?

Reply:
yes I made the changes